<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Optics_Dialog</class>
 <widget class="QDialog" name="Optics_Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>a</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>b</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>c</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>d</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
